interfacer/go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/browsh-org/browsh/interfacer
+
+go 1.14
```

Comment:
Is this version an issue? What minimum version of Go is being targeted?

Reply:
I never encountered any version related issues. I just pinned a version for the sake of pinning, in case there were unforeseeable issues with future versions. If latest Go works, then let's bump.